src/utils/network.ts

```diff
@@ -6,3 +9,18 @@ export function isForeignNetworkToken(defuseAssetId: string): boolean {
   }
   return false
 }
+
+export function networkLabelAdapter(defuseAssetId: string) {
+  const result = parseDefuseAsset(defuseAssetId)
+  const blockchain = result?.blockchain ?? ""
+  switch (blockchain) {
+    case BlockchainEnum.Near:
+      return "Near"
+    case BlockchainEnum.Eth:
+      return "Ethereum"
+    case BlockchainEnum.Btc:
+      return "Bitcoin"
```

Comment:
We can move names to constants, but not a big thing

Reply:
We use TypeScript for such cases and this is exactly our declaration of available types which more clear to read and use